### PR TITLE
rustdoc: Rename internal uses of `spotlight`

### PR DESCRIPTION
**Pull request by camelid**
I didn't make these renames in #80965 because I didn't want the PR to
conflict with #80914.

### CODE REVIEW

**Comment by camelid** [comment]
There are conflicts, but I'll wait to fix them until after #80965 lands. I mainly opened this PR so I don't forget to do the renames 😄

**Comment by camelid** [comment]
Rebased because #80965 has now merged.

File: src/librustdoc/html/static/rustdoc.css [hunk 1]
```diff
@@ -173,13 +173,10 @@ code, pre, a.test-arrow {
 	border-radius: 3px;
 	padding: 0 0.1em;
 }
-.docblock pre code, .docblock-short pre code, .docblock code.spotlight {
+.docblock pre code, .docblock-short pre code {
```
Comment (GuillaumeGomez) [inline src/librustdoc/html/static/rustdoc.css:176]: Shouldn't it be:

```suggestion
.docblock pre code, .docblock-short pre code, .docblock code.notable {
```

?

File: src/librustdoc/html/static/rustdoc.css [hunk 2]
```diff
@@ -173,13 +173,10 @@ code, pre, a.test-arrow {
...
 	padding: 0;
 	padding-right: 1ex;
 }
-.docblock code.spotlight :last-child {
```
Comment (GuillaumeGomez) [inline src/librustdoc/html/static/rustdoc.css:180]: Same here.

File: src/librustdoc/html/static/rustdoc.css (hunk 1 above)
Comment (camelid) [reply src/librustdoc/html/static/rustdoc.css:176]: See the commit message in 1fe0fe4. The classes appeared to be unused.

File: src/librustdoc/html/static/rustdoc.css (hunk 2 above)
Comment (camelid) [reply src/librustdoc/html/static/rustdoc.css:180]: https://github.com/rust-lang/rust/pull/83756#discussion_r606365491

**Comment by GuillaumeGomez** [comment]
You're right: both CSS classes are now unused. Then it's all good for me, thanks!

@bors: r+ rollup

**Pull request merged**